### DISCUSSION TITLE
截图用时400多ms是否正常，如何加速

### DISCUSSION THREAD

**Discussion by tkkcc** [MaaAssistantArknights/MaaAssistantArknights#8300] [Q&A]
提示截图用时400多ms，自动肉鸽可能出现异常
试了下mumu12和雷电9都是，1280x720分辨率，4核4G，宿主机是16G内存i5-10210U
这样的耗时是否正常，最佳截图耗时可以到多少呢，大家会遇到这种提示吗

**Comment by Rbqwow** [comment]
能用就不需要在乎那个
只要自动战斗正常跑起来了就完事
这东西你反复测几次都未必一样
